UE4SS/src/GUI/LiveView.cpp

```
@@ -1817,8 +1834,25 @@ namespace RC::GUI
         Output::send(STR("Searching by name...\n"));
         s_name_search_results.clear();
         s_name_search_results_set.clear();
+
+        uintptr_t address_to_search_by = 0;
+        if (UE4SSProgram::settings_manager.General.SearchByAddress &&
+           (LiveView::s_name_to_search_by.size() >= 6 && LiveView::s_name_to_search_by.size() <= 16))
+        {
+            try
+            {
+                address_to_search_by = std::stoull(LiveView::s_name_to_search_by, nullptr, 16);
+                Output::send<LogLevel::Default>(STR("Searching for object with address: {:016X}"), address_to_search_by);
+            }
+            catch (...) {}
```

Let's not fail silently.
Could you add a popup window so that the user is made aware immediately of the error.
We do that if a user tries to open a nullptr object, and you can search for `PropertyOpenNullptrWarning` for reference.